backend/bang-ggood/src/test/java/com/bang_ggood/checklist/ChecklistFixture.java

```diff
@@ -182,4 +189,5 @@ public class ChecklistFixture {
     );
 
     public static final ChecklistLike CHECKLIST_LIKE_1 = new ChecklistLike(CHECKLIST1_USER1);
```

Comment:
```suggestion
    public static final ChecklistLike CHECKLIST1_LIKE_1 = new ChecklistLike(CHECKLIST1_USER1);
```
어떤 체크리스트의 좋아요인지 네이밍에 드러나면 좋을 것 같아요!